res/controllers/Reloop-Digital-Jockey-2-IE.midi.xml

```diff
@@ -0,0 +1,1243 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.1.0+" schemaVersion="1">
+    <info>
+        <name>Reloop Digital Jockey 2 Interface Edition</name>
+        <author>DJ aK</author>
+        <description>Controller mapping for Reloop Digital Jockey 2 Interface Edition</description>
+        <forums>https://mixxx.discourse.group/t/reloop-digital-jockey-2-mapping-by-dj-ak/23971</forums>
+        <wiki></wiki>

```

Comment:
This line can be removed, not neccessary for complete empty elements

Reply:
the link to the manual can go here, if we merge mixxxdj/manual#468 before

Reply:
```suggestion
        <manual>reloop_digital_jockey_2_interface_edition</manual>
```